components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/AbstractAPIManager.java

```diff
@@ -28,14 +28,7 @@
 import org.json.simple.parser.JSONParser;
 import org.json.simple.parser.ParseException;
 import org.wso2.carbon.CarbonConstants;
-import org.wso2.carbon.apimgt.api.APIManagementException;
-import org.wso2.carbon.apimgt.api.APIManager;
-import org.wso2.carbon.apimgt.api.APIMgtResourceAlreadyExistsException;
-import org.wso2.carbon.apimgt.api.APIMgtResourceNotFoundException;
-import org.wso2.carbon.apimgt.api.ApplicationNameWhiteSpaceValidationException;
-import org.wso2.carbon.apimgt.api.ApplicationNameWithInvalidCharactersException;
-import org.wso2.carbon.apimgt.api.BlockConditionNotFoundException;
-import org.wso2.carbon.apimgt.api.PolicyNotFoundException;
+import org.wso2.carbon.apimgt.api.*;
```

Comment:
Let's keep separate classes instead of *

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/11469/commits/f23c8e152f9f622508e0d875cb5d304c3e640ed5